startup crash

**Describe the bug**
Upon starting the session is shown the message "Oops, Something Went Wrong" and in the Log of the Possible Startup Crash terminal

**To Reproduce**
Steps to reproduce the behavior:
Upon entering a new session at html5cliente already occurs the error

**Screenshots**
<img width="1366" height="741" alt="Image" src="https://github.com/user-attachments/assets/1733232d-3dbc-4561-8dad-ef7c3a5b026a" />

**BBB version:**
BigBlueButton Server 3.0.11 (2829)

**Additional context**
```
[10:14:41:0497]  INFO: clientLogger: Fetching GraphQL URL e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.m.js:1 [10:14:41:0506]  INFO: clientLogger: Connecting to GraphQL server e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.m.js:1 [10:14:41:0507]  INFO: clientLogger: Fetching locale e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.m.js:1 [10:14:41:0719]  INFO: clientLogger: Fetching GraphQL URL e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.m.js:1 [10:14:41:0719]  INFO: clientLogger: Connecting to GraphQL server e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.m.js:1 [10:14:41:0720]  INFO: clientLogger: Connecting to GraphQL server e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.m.js:1 [10:14:42:0552]  INFO: clientLogger: Connection status changed to true e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.ts:112 [10:14:44:0083]  INFO: clientLogger: User joined main room e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
hook.js:451  Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at C (focus-trap.esm.js:387:13)
    at Object.activate (focus-trap.esm.js:835:9)
    at d.value (focus-trap-react.js:299:28)
    at d.value (focus-trap-react.js:311:14)
    at wl (react-dom.production.min.js:260:217)
    at bl (react-dom.production.min.js:259:446)
    at yl (react-dom.production.min.js:258:431)
    at react-dom.production.min.js:282:96
    at Ac (react-dom.production.min.js:280:398)
    at lc (react-dom.production.min.js:272:439)
overrideMethod @ hook.js:451
ds @ react-dom.production.min.js:188
(anônimo) @ react-dom.production.min.js:189
Wo @ react-dom.production.min.js:156
wl @ react-dom.production.min.js:260
bl @ react-dom.production.min.js:259
yl @ react-dom.production.min.js:258
(anônimo) @ react-dom.production.min.js:282
Ac @ react-dom.production.min.js:280
lc @ react-dom.production.min.js:272
Hi @ react-dom.production.min.js:127
(anônimo) @ react-dom.production.min.js:282
Ac @ react-dom.production.min.js:280
ic @ react-dom.production.min.js:269
k @ scheduler.production.min.js:13
P @ scheduler.production.min.js:14
component.jsx:52 ErrorBoundary caught an error:  Error: Your focus-trap must have at least one container with at least one tabbable node in it at all times
    at C (focus-trap.esm.js:387:13)
    at Object.activate (focus-trap.esm.js:835:9)
    at d.value (focus-trap-react.js:299:28)
    at d.value (focus-trap-react.js:311:14)
    at wl (react-dom.production.min.js:260:217)
    at bl (react-dom.production.min.js:259:446)
    at yl (react-dom.production.min.js:258:431)
    at react-dom.production.min.js:282:96
    at Ac (react-dom.production.min.js:280:398)
    at lc (react-dom.production.min.js:272:439)
component.jsx:53 ErrorInfo:  {componentStack: '\n    at d (https://bbb23.saeteorico.com.br/html5cl…dle.ac736d2afa5bfc2a4194.js:2:2562532)\n    at rJt'}
index.ts:104  [10:14:44:0717] ERROR: clientLogger: Possible startup crash e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158472)
overrideMethod @ hook.js:451
e.write @ index.m.js:1
t._emit @ index.m.js:1
(anônimo) @ index.m.js:1
value @ index.ts:104
value @ component.jsx:54
wl @ react-dom.production.min.js:260
bl @ react-dom.production.min.js:259
yl @ react-dom.production.min.js:258
(anônimo) @ react-dom.production.min.js:282
Ac @ react-dom.production.min.js:280
lc @ react-dom.production.min.js:272
Hi @ react-dom.production.min.js:127
(anônimo) @ react-dom.production.min.js:282
Ac @ react-dom.production.min.js:280
ic @ react-dom.production.min.js:269
k @ scheduler.production.min.js:13
P @ scheduler.production.min.js:14
component.jsx:900  Uncaught (in promise) ReferenceError: bbbVideoStream is not defined
    at t.<anonymous> (component.jsx:900:36)
    at d (component.jsx:2:1)
    at Generator.<anonymous> (component.jsx:2:1)
    at Generator.next (component.jsx:2:1)
    at aT (component.jsx:2:1)
    at a (component.jsx:2:1)
(anônimo) @ component.jsx:900
d @ component.jsx:2
(anônimo) @ component.jsx:2
(anônimo) @ component.jsx:2
aT @ component.jsx:2
a @ component.jsx:2
Promise.then
aT @ component.jsx:2
a @ component.jsx:2
Promise.then
aT @ component.jsx:2
a @ component.jsx:2
Promise.then
aT @ component.jsx:2
a @ component.jsx:2
(anônimo) @ component.jsx:2
(anônimo) @ component.jsx:2
(anônimo) @ component.jsx:845
value @ component.jsx:760
i @ component.jsx:321
Promise.then
value @ component.jsx:375
wl @ react-dom.production.min.js:260
bl @ react-dom.production.min.js:259
yl @ react-dom.production.min.js:258
(anônimo) @ react-dom.production.min.js:282
Ac @ react-dom.production.min.js:280
ic @ react-dom.production.min.js:269
k @ scheduler.production.min.js:13
P @ scheduler.production.min.js:14
index.ts:112 [10:14:49:0685]  INFO: clientLogger: Connection terminated (4499) e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
2index.ts:112 [10:14:49:0686]  INFO: clientLogger: Connection terminated (4499) e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.ts:112 [10:14:59:0686]  INFO: clientLogger: Connection status changed to false e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756)
index.ts:112 [10:15:00:0499]  INFO: clientLogger: Connection status changed to true e.value (https://bbb23.saeteorico.com.br/html5client/bundle.ac736d2afa5bfc2a4194.js:2:158756) ```

--- a/bigbluebutton-html5/imports/ui/components/common/modal/simple/component.jsx
+++ b/bigbluebutton-html5/imports/ui/components/common/modal/simple/component.jsx
@@ -139,7 +139,7 @@ class ModalSimple extends Component {
         style={modalStyles}
         {...otherProps}
       >
-        <FocusTrap active={modalIsOpen} focusTrapOptions={{ initialFocus: false }}>
+        <FocusTrap active={modalIsOpen} focusTrapOptions={{ initialFocus: false, fallbackFocus: '#fallback-element' }}>
           <div ref={this.modalRef}>
             <Styled.Header
               hideBorder={hideBorder}
@@ -156,6 +156,7 @@ class ModalSimple extends Component {
             </Styled.Header>
             <Styled.Content>
               {children}
+              <div id="fallback-element" tabIndex="-1" />
             </Styled.Content>
           </div>
         </FocusTrap>

